src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -400,11 +400,21 @@
                     
                     <te:SectionControl.Content>
                         <Grid>
+                            <Grid.Resources>
+                                <!-- We have to do contortions to get to the PullRequestDetailViewModel which contains the OpenFile and DiffFile
+                                     commands: the control which hosts the context menu must have the PullRequestDetailViewModel as its tag,
+                                     which is then set as the tag on the ContextMenu which is finally reacable from the MenuItem. -->
+                                <ContextMenu x:Key="FileContextMenu" Tag="{Binding PlacementTarget.Tag, RelativeSource={RelativeSource Mode=Self}}">
```

Comment:
see 8fb4deaf0e74e5f11a73b8c59719237fdb092346 for how I've got it to work with out the nasty tag hack that `ContextMenu` requires otherwise.